lib/phase/showTrialEvaluation.m

```diff
@@ -0,0 +1,120 @@
+function [outcomeKind, outcomeLevel] = showTrialEvaluation(trial, config)
+% Allow users to call this function from wherever they want 
+% Be able to receive the trial specification, block configuration. 
+
+% Evaluate the trial: adapted lines 92-end of pickAndEvaluateTrial.
+%% 2. Evaluate the trial
+% summary = experimentSummary(block, blockIdx, trialIdx);
+[ outcomeKind, outcomeLevel, outcomeColorIdx, trueProb, randDraw ] = evaluateTrial(trial);
+
+%% 3. Draw the trial
+% 0. Open PTB window if it isn't yet opened (`isempty(Screen('Windows'))`)
+standalone = false;
+if isempty(Screen('Windows'))
+  standalone = true;
+  config = loadPTB(config);
+  config.runSetup.textures = loadTexturesFromConfig(config);
+end
+
+
+% 1. Use drawLotto and drawRef to show the lotto
+config.task.fnHandles.bgrDrawFn(config, trial);
+drawLotto(trial, config);
+config.task.fnHandles.referenceDrawFn(config, trial);
+Screen(config.device.windowPtr, 'Flip');
+% 2. Disappear the non-picked choice
+WaitSecs(5);
+switch trial.choseLottery
+  case 0
+    config.task.fnHandles.referenceDrawFn(config, trial);
+    msg = 'You chose the certain option.';
+  
+  case 1
+  windowPtr = config.device.windowPtr;
+    % Determine message to display
+    
+    % For monetary gamble 
+    if strcmp(config.runSetup.blockName, "Monetary") == 1   
```

Comment:
This will not work on Matlab versions prior to 2016b be — don't ask me why, but Matlab was very strict that strings are denoted by apostrophes, not by quotes. 

Also, note that this code doesn't create a basecase for a block whose name is neither monetary nor medical, which means that it will fail hard in those cases.